pkg/virt-handler/cgroup/cgroup.go

```diff
@@ -91,7 +92,7 @@ func managerPath(taskPath string) string {
 
 // NewManagerFromPid initializes a new cgroup manager from VMI's pid.
 // The pid is expected to VMI's pid from the host's viewpoint.
-func NewManagerFromPid(pid int) (manager Manager, err error) {
+func NewManagerFromPid(pid int, deviceRules []*devices.Rule) (manager Manager, err error) {
```

Comment:
Maybe we can get the whole `Resources` struct instead?

Reply:
Not sure. Current implementation deals only with the device rules. Passing the bigger struct will IMHO make the interface (the function signature) more confusing. I believe this can easily be changed in the future when (and if) we start handling something else from the `Resource` struct.